src/Sulu/Bundle/AdminBundle/Resources/js/components/RectangleSelection/tests/__snapshots__/ModifiableRectangle.test.js.snap

```diff
@@ -0,0 +1,55 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`The component should render 1`] = `
+<ModifiableRectangle
```

Comment:
Have you used the wrong method for this snapshot? Because it's again not rendering to plain html.